browser/js/app.js

```diff
@@ -73,19 +66,34 @@ var player = {
 
 var otherPlayers = [];
 
+socket.on('otherPlayerJoin', function (otherPlayerData) {
+    console.log(otherPlayerData.id + ' has joined!');
+    otherPlayerData.sprite = new Sprite('img/capguy-walk.png', [0, 0], [184, 325], 16, [0, 1, 2, 3, 4, 5, 6, 7]);
+    otherPlayers.push(otherPlayerData);
```

Comment:
augmented this listener to also add that other player that just joined to my other players array